presto-accumulo/src/main/java/com/facebook/presto/accumulo/io/AccumuloPageSinkProvider.java

```diff
@@ -52,15 +54,16 @@ public AccumuloPageSinkProvider(
     }
 
     @Override
-    public ConnectorPageSink createPageSink(ConnectorTransactionHandle transactionHandle, ConnectorSession session, ConnectorOutputTableHandle outputTableHandle)
+    public ConnectorPageSink createPageSink(ConnectorTransactionHandle transactionHandle, ConnectorSession session, ConnectorOutputTableHandle outputTableHandle, PageSinkProperty pageSinkProperty)
```

Comment:
Lets rename the `PageSinkProperty` to `PageSinkProperties`. Same for the parameter names.